src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -859,7 +859,7 @@ static int zocl_drm_platform_probe(struct platform_device *pdev)
 	 * For PR platform, the FPGA manager is required. No good way to
 	 * determin if it is a PR platform at probe.
 	 */
-	fnode = of_get_child_by_name(of_root,
+	fnode = of_find_node_by_name(of_root,
```

Comment:
Just a reminder that this change will impact versal platform as well, VCK5000 and VCK190, where the fgpa_driver_name would be versal-fpga. But took a quick look at the versal-fpga node on versal, we should be fine. But it would be nice if we can give it a test.

Reply:
@larry9523 , verified on VCK190. I dont have VCK5000 board, checking whether anyone has it in XHD.